maas/meta/backupninja.yml

```diff
@@ -0,0 +1,6 @@
+backup:
+  maas:
+    fs_includes: []
+    fs_excludes: []
+    actions:
+    - cmd: "rsync -aH  --exclude='/var/lib/maas/boot-resources' --relative /etc/maas /var/lib/maas /var/backups/maas_config"
```

Comment:
otherwise I wont be able to store it in /var/backups/maas_config where I'm mounting glusterfs volume